packages/web/nexus/subscription.ts

```diff
@@ -0,0 +1,322 @@
+import { MembershipSubscriptionPeriod, InputJsonValue, PrismaClient } from '@journaly/j-db-client'
+import {
+  arg,
+  booleanArg,
+  extendType,
+  objectType,
+  stringArg,
+} from '@nexus/schema'
+import stripe from './utils/stripe'
+
+const MembershipSubscription = objectType({
+  name: 'MembershipSubscription',
+  definition(t) {
+    t.model.id()
+    t.model.period()
+    t.model.userId()
+    t.model.expiresAt()
+    t.model.cancelAtPeriodEnd()
+    t.model.lastFourCardNumbers()
+    t.model.cardType()
+  }
+})
+
+const getSubscriptionPriceId = (subType: MembershipSubscriptionPeriod) => {
+  switch(subType) {
+    case MembershipSubscriptionPeriod.MONTHLY:
+      return 'price_1ISRgvB8OEjVdGPaQr7ZANW8'
```

Comment:
If we're settled on the prices, let's copy them to prod in stripe and see if they maintain the same ID. If not we'll probably want to pull these out into env vars

Reply:
We still need to settle on the 3-month and 1-year prices since the new discussions where we increased the base monthly price to £12 (although I think we still need to finalise that, too